packages/cardano-services/test/cli.test.ts

```diff
@@ -65,6 +65,18 @@ const assertServiceHealthy = async (
   else expect(res.data).toEqual(healthCheckResponse);
 };
 
+const assertCORSHeaders = async (apiUrl: string, origin: string) => {
+  expect.assertions(1);
+  await serverReady(apiUrl);
+  const headers = { 'Content-Type': 'application/json', Origin: origin };
+  try {
+    const res = await axios.get(`${apiUrl}/health`, { headers });
+    expect(res.headers['access-control-allow-origin']).toEqual(origin);
+  } catch (error) {
+    expect((error as AxiosError).response?.status).toBe(404);
```

Comment:
We may get false positives with current implementation, as caller doesn't know which assertion was made
```suggestion
const assertCORSHeaders = async (apiUrl: string, origin: string, expectedFailureStatus?: number) => {
  expect.assertions(1);
  await serverReady(apiUrl);
  const headers = { 'Content-Type': 'application/json', Origin: origin };
  try {
    const res = await axios.get(`${apiUrl}/health`, { headers });
    expect(res.headers['access-control-allow-origin']).toEqual(origin);
  } catch (error) {
    expect((error as AxiosError).response!.status).toBe(expectedFailureStatus);
```